.babelrc

```diff
@@ -1,11 +1,20 @@
 {
 	"presets": [
-		"es2015",
+		[ "env", {
+			"targets": {
+				"browsers": [
+					"last 2 versions",
+					"Safari >= 9",
+					"iOS >= 9",
+					"not ie <= 10",
+					"> 1%"
```

Comment:
It includes the following browsers listed [here](http://browserl.ist/?q=%3E1%25):

<img width="948" alt="_1_" src="https://user-images.githubusercontent.com/4044428/28981266-4ef475ae-790e-11e7-83ea-710f9b738355.png">

I think it'll be fine to drop, so I'll go ahead and get that taken care of.